docs/transpile/custom-transpiler-pass.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing your own Pass\n",
+    "\n",
+    "In the [Transpiler stages](transpiler-stages) topic we went over the default behavior of the transpiler and the various stages that compose a transpilation pipeline.  Qiskit allows you to create custom transpilation workflows using: the `PassManager` and `StagedPassMager` objects, writing your own bespoke transpiler passes, and  Here we will take a more careful look at the `PassManager` and `StagedPassManager` objects to customize which passes are executed at what stage and walk through a few examples.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## DAG Representation\n",
+    "\n",
+    "\n",
+    "However, before we start it is important to first introduce the internal representation of quantum circuits in Qiskit, the *Directed Acyclic Graph* or **DAG**.  If you want to follow along here, you will need to install the `pydot` library and the `graphivz` library for the DAG plotting functions.  Use the python package manager of your choice (such as `pip` or `conda`) to install `pydot` and your system's native package manager (e.g. `apt`, `brew`, `yum`, `dnf`, etc.) for `graphivz`.\n",
+    "\n",
+    "\n",
```

Comment:
There is a intro tutorial on dags I wrote for the rustworkx docs here: https://qiskit.org/ecosystem/rustworkx/tutorial/dags.html it might be a good thing to link to.

Reply:
I think the ecosystem links are moving to... happy to link to it when it finds a new home.

Reply:
Added the link!